package.json

```diff
@@ -125,6 +130,7 @@
     "globals": "^16.0.0",
     "happy-dom": "^17.4.7",
     "jiti": "^2.4.2",
+    "jsdom": "^26.1.0",
```

Comment:
We should aim later that we have these dependencies on `devDependencies`, generally testing libs, types and so on. Can make dedicated task on that, although there is a task specifically for checking dependencies across workspace.